src/app/features/trips/models/order.model.ts

```diff
@@ -0,0 +1,16 @@
+export interface Order {
+    id: number,
+    rideId: number,
+    routeId: number,
+    seatId: number,
+    userId: number,
+    status: 'active' | 'completed' | 'rejected' | 'canceled',
+    path: number[],
+    carriages: string[],
+    schedule: {
+        segments: {
+            time: [string, string],
+            price: { [key: string]: number }
+        }[]
+    }
```

Comment:
Move this to separate `schedule` and `segments` interface. It's best practice, cause in case our app will contuinue to grow, new elements in our requests can appear and we will need to refactor this anyway)